tests/drivers/build_all/modem_ublox_sara.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_TEST=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_NETWORKING=y
```

Comment:
This will enable `MODEM_HL7800` for the `pinnacle_100_dvk` board. Either need to add `NET_IPV4` to this config or exclude `pinnacle_100_dvk` from `drivers.modem.ublox_sara.build`.

Reply:
I just excluded it, seems easier.